docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,142 @@
+
+Important is that every step is independent of each other. A Managed Kubernetes service that implements _Cluster Maintenance_ doesn't necessarily already implement _Logs_.
+That in turn also means we must allow some leeway for each Managed Kubernetes service in respect to the requirements, should a service not allow for a feature or makes it overly complex.
+A Managed Kubernetes service is allowed to deviate from a requirement when it makes sense and can be documented and reasoned about accordingly.
+
+To understand the personas referenced in this document, see xref:references/glossary.adoc#_personas[Glossary - Personas].
+
+== Initial Release: Cluster Provisioning
+
+As an *End-User* I can:
+
+* provision a Cloud Kubernetes environment on our supported Cloud Service Providers, fully self-service.
+* connect to the Cloud Kubernetes API for using it.
+* customize my cluster with a pre-defined set of features and components.
+** scale my cluster horizontally and vertically, as my needs change.
+** choose the https://products.docs.vshn.ch/products/service_levels.html[service level^] of the instance.
+* delete the Cloud Kubernetes environment again.
+
+As a *Technical VSHNeer* I can:
+
+* see and configure the https://products.docs.vshn.ch/products/service_levels.html[service level^] of the instance.
+* automatically bill the service to the end-user according to the specified price model.
+
+== Iteration: Cluster Maintenance
+
+As an *End-User* I can:
+
+* schedule maintenance windows for my Cloud Kubernetes environment, during which updates, patches, or other changes can be applied automatically.
+* receive notifications in advance of any scheduled maintenance, allowing me to plan accordingly and to minimize any disruption to my applications.
+
+As a *Technical VSHNeer* I can:
+
+* monitor the status of the Cloud Kubernetes environment during the maintenance window, to ensure that the maintenance is operating as expected.
+
+== Iteration: Cluster Version Upgrade
+
+As an *End-User* I can:
+
+* schedule an automated upgrade of my Cloud Kubernetes environment to the next minor version.
+
+As a *Technical VSHNeer* I can:
+
+* trigger an automated upgrade of the Cloud Kubernetes environment to the next minor version.
+
+== Iteration: Cluster Backup and Restore
+
+As an *End-User* I can:
+
+* rely on automated backups of my Cloud Kubernetes environment configuration and cluster state.
+* schedule backups cluster state backups to run automatically, at a time and frequency that suits my needs.
+* configure data retention policies to ensure that backups are kept for the desired length of time.
+* quickly and easily restore my Cloud Kubernetes environment to a previous state, including all cluster configuration.
+* test restores in a test environment before performing a full restore, to ensure that the backup is fully restorable.
```

Comment:
That is a real tough one. It probably needs clarification of scope. This can read as _restore a cluster along with everything deployed on it_. Just a few points where this might go wrong:

* PVs can only be bound to one cluster or might not be available anymore
* Deployments might reference external resources (S3, Cloud databases, APIs). So the copy of an application might start to fight about state with the actual production one)
* And probably a lot more

Reply:
I strongly agree. I'd even say: Let's remove that one for now, it could make this iteration into a state where it isn't possible to get it done without a huge amount of effort.